dist-build/android-armv7-a.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-export TARGET_ARCH=armv7-a
-export CFLAGS="-Os -mfloat-abi=softfp -mfpu=vfpv3-d16 -mthumb -marm -march=${TARGET_ARCH}"
-ARCH=arm HOST_COMPILER=armv7a-linux-androideabi "$(dirname "$0")/android-build.sh"
+export CFLAGS="-Os -mfloat-abi=softfp -mfpu=vfpv3-d16 -mthumb -marm -march=armv7a"
+HOST_COMPILER=armv7a-linux-androideabi TARGET_ARCH_DIR=arm-linux-androideabi "$(dirname "$0")/android-build.sh"
```

Comment:
`HOST_COMPILER` is not very self-explanatory. One may expect to set it to something such as `clang` or `zig cc`. Maybe something like `NDK_COMPILER_DIR`?

Reply:
I'm happy to change it the variable name comes from the version before my edits. it's HOST_COMPILER because it's the name of the compiler on the host that's used for the cross compilation. It's actually a program so NDK_COMPILER would make more sense than NDK_COMPILER_DIR.

Reply:
NDK_COMPILER also isn't technically correct as it's only part of the compiler's name NDK_COMPILER_PREFIX might make more sense though it might also be a different kind of confusing... My preference is still with NDK_COMPILER but I don't have strong feelings about this item and will go with whatever you're happy with.